frontend/src/components/facilities/FacilityInfo.vue

```diff
@@ -109,6 +109,77 @@
           </v-col>
         </v-row>
         <v-divider />
```

Comment:
should we move this v-divider to under the expansion-panel?

Reply:
and maybe add some padding top/bottom so it is more visible

Reply:
I'm just going to hide the divider if we have the expansion panel... I don't feel like we really need it in that case